src/Node.fs

```diff
@@ -3279,7 +3287,47 @@ module tls =
 
     type [<AllowNullLiteral>] TLSSocketStatic =
         /// Construct a new tls.TLSSocket object from an existing TCP socket.
-        [<Emit "new $0($1...)">] abstract Create: socket: net.Socket * ?options: obj -> TLSSocket
+        [<Emit "new $0($1...)">] abstract Create: socket: net.Socket * ?options: TLSSocketStaticOptions -> TLSSocket
```

Comment:
I dropped the `Create` here for the constructor. `TLSSocketStaticOptions` sounds good. Should it be `TLSSocketStaticCreateOptions` though. Don't think so.